### PR TITLE
Containers

### PR DESCRIPTION
**Pull request by MaxUlysse**
Many thanks to contributing to nf-core/rnafusion!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/rnafusion branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/rnafusion)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by matq007** [comment]
Let's keep the environment names consistent with container names. Bring back the `*_v*` 💯

**Comment by MaxUlysse** [comment]
Just did the reverse and remove the `*_v*` ;-)
Make more sense I believe, we don't have it for the main container actually since it's not there by default, so let's remove it for all

**Comment by matq007** [comment]
ok so now we have to rebuild all images and re-upload them because the names on DockerHub are different, example: `docker pull nfcore/rnafusion:squid_v1.5-star2.7.0f`

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * `MultiQC v1.7`
   * `STAR aligner v2.7.0f`
 
-### Added
+### Added - [1.1.0]
```
Comment (matq007) [inline CHANGELOG.md:32]: Why adding version? This is not following the guidelines.

Comment (MaxUlysse) [reply CHANGELOG.md:32]: I had mardownlint issues, but now I know that it was coming from another place.
removing all that.

**Comment by MaxUlysse** [comment]
finally got this linting to work again...
Sorry about all that

**Pull request merged**